src/backend/PortabilityService.Gateway/appsettings.json

```diff
@@ -0,0 +1,269 @@
+        "QoSOptions": {
+          "ExceptionsAllowedBeforeBreaking": 3,
+          "DurationOfBreak": 10000,
+          "TimeoutValue": 5000
+        }
+      },
+      {
+        "RouteDescription": "Data Update Service Redirect",
+        "DownstreamPathTemplate": "/api/data",
+        "DownstreamScheme": "https",
+        "DownstreamHostAndPorts": [
+          {
+            "Host": "portability.dot.net",
+            "Port": 443
+          }
+        ],
+        "UpstreamPathTemplate": "/api/data",
+        "UpstreamHttpMethod": [ "PUT" ],
+        "RateLimitOptions": {
+          "ClientWhitelist": [],
+          "EnableRateLimiting": true,
+          "Period": "10s",
+          "PeriodTimespan": 10,
+          "Limit": 10
+        },
+        "QoSOptions": {
+          "ExceptionsAllowedBeforeBreaking": 3,
+          "DurationOfBreak": 10000,
+          "TimeoutValue": 60000
+        }
+      },
+
+      {
+        "RouteDescription": "ResultFormat Service Redirect",
+        "DownstreamPathTemplate": "/api/resultformat",
+        "DownstreamScheme": "https",
+        "DownstreamHostAndPorts": [
+          {
+            "Host": "portability.dot.net",
+            "Port": 443
+          }
+        ],
+        "UpstreamPathTemplate": "/api/resultformat",
+        "UpstreamHttpMethod": [ "GET" ],
+        "FileCacheOptions": { "TtlSeconds": 5 },
+        "QoSOptions": {
+          "ExceptionsAllowedBeforeBreaking": 3,
+          "DurationOfBreak": 10000,
+          "TimeoutValue": 5000
+        }
+      },
+      {
+        "RouteDescription": "Default ResultFormat Service Redirect",
+        "DownstreamPathTemplate": "/api/resultformat/default",
+        "DownstreamScheme": "https",
+        "DownstreamHostAndPorts": [
+          {
+            "Host": "portability.dot.net",
+            "Port": 443
+          }
+        ],
+        "UpstreamPathTemplate": "/api/resultformat/default",
+        "UpstreamHttpMethod": [ "GET" ],
+        "FileCacheOptions": { "TtlSeconds": 5 },
+        "QoSOptions": {
+          "ExceptionsAllowedBeforeBreaking": 3,
+          "DurationOfBreak": 10000,
+          "TimeoutValue": 5000
+        }
+      },
+
+      {
+        "RouteDescription": "Target Service Redirect",
+        "DownstreamPathTemplate": "/api/target",
+        "DownstreamScheme": "https",
+        "DownstreamHostAndPorts": [
+          {
+            "Host": "portability.dot.net",
+            "Port": 443
+          }
+        ],
+        "UpstreamPathTemplate": "/api/target",
+        "UpstreamHttpMethod": [ "GET" ],
+        "FileCacheOptions": { "TtlSeconds": 5 },
+        "QoSOptions": {
+          "ExceptionsAllowedBeforeBreaking": 3,
+          "DurationOfBreak": 10000,
+          "TimeoutValue": 5000
+        }
+      }
+    ],
+    "GlobalConfiguration": {
+      "RequestIdKey": "Correlation-Id"
+    }
+  },
+  "Serilog": {
+    "MinimumLevel": "Warning",
+    "WriteTo": [
+      {
+        "Name": "Console",
+        "Args": {
```

Comment:
We probably don't want to log to Console in production.

Reply:
I think as long as the logging is minimal (warning, in this case), console logging in production is ok. A lot of orchestrators will collect standard out from their services and make it available.

The console logger used to be quite slow (so this would be bad), but that was fixed in 2.0. And this is Serilog console logging, anyhow, which doesn't have the same perf problem.